src/main/java/core/basesyntax/service/activity/ActivityHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.activity;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+
+public interface ActivityHandler {
+    FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
```suggestion
```
don't think this is a good idea, having dao as a constant in interface. Better pass it as an argument to methods